### ISSUE TITLE
Etcd metrics banner are not working

### ISSUE THREAD

**Issue by NRCan-LGariepy** [rancher/dashboard#11579]
Version: RKE2, v2.8.3

**Describe the bug**
The UI does not accurately reflect the state of etcd

**Result**
Etcd reports that there are no leaders, no elections, etc.

**Expected Result**
I expect the UI to accurately reflect the state of etcd.

**Screenshots**
![etcd](https://github.com/user-attachments/assets/6923c1f7-daa2-468d-8b80-c9b743c069f2)
![etcd2](https://github.com/user-attachments/assets/a1b3a228-7c0e-4955-9e77-b165a54c63ab)

**Comment by richard-cox** [comment]
The UI get's this info via grafana 
```
const end = Date.now() / 1000;
const start = end - (5 * 60)
query=max(etcd_server_has_leader)&start=${ start }&end=${ end  }&step=30
```

@NRCan-LGariepy Can you confirm the version of the `Monitoring` chart you have installed?

**Comment by skanakal** [comment]
@richard-cox seems its duplicate of https://github.com/rancher/dashboard/issues/10172

**Comment by NRCan-LGariepy** [comment]
@richard-cox I can confirm that the query you're proposing works:

```bash
# Step 1: Port-forward
 kubectl port-forward --namespace cattle-monitoring-system service/rancher-monitoring-prometheus 9090

# Step 2: (In another terminal) curl the endpoint
curl 'http://localhost:9090/api/v1/query_range?query=max(etcd_server_has_leader)&start=1723034894&end=1723035194&step=30s'
```

This gives the expected result (i.e. the value 1 at all queried steps).

The chart version is 103.1.1+up45.31.1

**Comment by richard-cox** [comment]
@NRCan-LGariepy Thanks for the update. It looks like this was fixed in 2.9.0. If on upgrade you still see the issue please let us know by re-opening this issue

**Comment by NRCan-LGariepy** [comment]
@richard-cox I can confirm that 2.9.0 did the trick. Cheers!
